components/bitcoin-topics/layouts/TopicBanner.tsx

```diff
@@ -0,0 +1,259 @@
+"use client"
+
+import { ReactNode, useState, useEffect } from "react"
+import { CoreContent } from "pliny/utils/contentlayer"
+import type { Topic, Authors } from "contentlayer/generated"
+import { Navigation } from "../Navigation"
+import { Hero } from "../hero/Hero"
+import { PrevNextLinks } from "../topic/PrevNextLinks"
+import { Prose } from "../topic/Prose"
+import { TopicHeader } from "../topic/TopicHeader"
+import { usePathname } from "next/navigation"
+import Image from "next/image"
+import Link from "next/link"
+
+import { motion, AnimatePresence } from "framer-motion"
+import { MoreVertical } from "lucide-react"
+import React from "react"
+import { GitHubIcon } from "@/public/images/topics-hero/GitHubIcon"
+
+interface LayoutProps {
+    content: CoreContent<Topic>
+    authorDetails: CoreContent<Authors>[]
+    next?: { path: string; title: string }
+    prev?: { path: string; title: string }
+    children: ReactNode
+}
+
+export default function TopicBanner({
+    content,
+    next,
+    prev,
+    children
+}: LayoutProps) {
+    const pathname: string | null = usePathname()
+    const isHomePage: boolean = pathname === "/topics"
+    const [isNavOpen, setIsNavOpen] = useState(false)
+    const [currentPath, setCurrentPath] = useState<string[]>([])
+    const [isScrolled, setIsScrolled] = useState(false)
+
+    const { title, tags }: { title: string; tags: string[] } = content
+
+    const toggleNav = () => setIsNavOpen(!isNavOpen)
+
+    const githubEditUrl = `https://github.com/jrakibi/bitcoin-topics/edit/main/topics/${content.slug}.mdx`
+
+    const EditOnGitHubButton = () => (
+        <a
```

Comment:
Is there any reason you use the <a></a> over the next/Link tag ?

Reply:
Generally I use the `a` tag for external links, since there are no benefits to using `Link`, such as client-side routing, unless our concern is maintaining consistency by using the same tag throughout the application